.github/workflows/profiling.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - uses: ./.github/actions/node/20
       - run: yarn test:profiler:ci
       - run: yarn test:integration:profiler
-      - uses: ./.github/actions/node/21
+      - uses: ./.github/actions/node/latest
```

Comment:
We do not have something for 21 ?

Reply:
Odd numbered versions are not LTS, and we do not test them except when they are `latest`

Reply:
Odd numbers go EOL to moment the next even numbered release happens, so we don't consider them supported at all, but we test against them because it can, to some extent, warn us ahead of time when an issue is coming. Generally more significant breaking changes are saved for odd releases so there's time to test them in userland and revert in the next even release. While even releases tend to only take more conservative breaking changes due to being harder to undo before it becomes LTS.